discord/__init__.py

```diff
@@ -69,6 +69,7 @@ class VersionInfo(NamedTuple):
     serial: int
 
 
-version_info: VersionInfo = VersionInfo(major=2, minor=0, micro=0, releaselevel='alpha', serial=0)
+version_info: VersionInfo = VersionInfo(
+    major=2, minor=0, micro=0, releaselevel='alpha', serial=0)
```

Comment:
already cool, it's don't need a new line
```suggestion
version_info: VersionInfo = VersionInfo(major=2, minor=0, micro=0, releaselevel='alpha', serial=0)
```

Reply:
I don't think we should change the version rn, cz EQ is still working on new things....

Reply:
wait nvm
my bad